### PR TITLE
Separate temporary cookie for adtest query string params starting with 'demo'

### PR DESCRIPTION
**Pull request by tijanamiletic**
For adtest query string parameters which start with 'demo', a separate cookie is set temporarily and deleted. This has been implemented so that the demo examples can only be viewed when there is an explicit string available in the URL. This change should not affect the persistence of other adtest cookies.

### CODE REVIEW

**Comment by johnduffell** [comment]
if you only want it for one request, it would be worth considering a URL hash rather than a cookie, if possible

**Comment by tijanamiletic** [comment]
I have now simplified to use the same cookie which is set anyway. The cookie is deleted if the string params starts with 'demo'.

**Comment by johnduffell** [comment]
I'm also nervous about methods called "get" that also delete stuff!

**Comment by tijanamiletic** [comment]
OK, good point, have renamed that function.

**Comment by johnduffell** [comment]
thanks @tijanamiletic , actually I don't know the architecture of this area of code, so maybe discuss with someone more experienced about the hash thing

File: static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js [hunk 1]
```diff
@@ -65,6 +65,16 @@ define([
             });
 
             return abParams;
+        },
+        adtestParams = function () {
+            if (cookies.get('adtest')) {
+                var cookieAdtest = cookies.get('adtest');
+                var first4Char = cookieAdtest.substring(0, 4);
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js]: use only one var and , between variables

**Comment by Calanthe** [comment]
:+1: looks good :)

**Pull request merged**